src/displayapp/screens/ScreenList.h

```diff
@@ -9,16 +9,33 @@
 namespace Pinetime {
   namespace Applications {
     namespace Screens {
+
+      enum class ScreenListModes {UpDown, RightLeft, LongPress};
       template <size_t N>
       class ScreenList : public Screen {
         public:
-          ScreenList(DisplayApp* app, std::array<std::function<std::unique_ptr<Screen>()>, N>&& screens)
-          : Screen(app), screens{std::move(screens)}, current{this->screens[0]()} {
 
+          ScreenList(            
```

Comment:
I'm not a huge fan of this indentation. Maybe apply the changes using the original indentation?